### PR TITLE
Add Cask for 'SafeSign Identity Client'

### PR DESCRIPTION
**Pull request by berniechiu**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

Ref: https://github.com/caskroom/homebrew-cask/issues/24722

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Before discussing inclusion of this further, I’d like @rogeriopradoj’s input here, since he knows the app/cask well.

**Comment by berniechiu** [comment]
alright I'll close this one first

File: Casks/safesign-driver.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'safesign-driver' do
```
Comment (vitorgalvao) [inline Casks/safesign-driver.rb]: Call it `safesign-identity-client` instead.

File: Casks/safesign-driver.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  version '3.0.93'
+  sha256 '9c1f955293f5cea03bdaa78bc34b3b728cbbfed7fb1a4308ab337b2b52a538fd'
+
+  # site.solutinet.com.br was verified as official when first introduced to the cask
+  url "http://site.solutinet.com.br/2014/images/files/instaladores-e-manuais/Safesign-#{version.no_dots}-MacOS-driver-manual.zip"
+  name 'SafeSign Driver'
```
Comment (vitorgalvao) [inline Casks/safesign-driver.rb]: Make it `SafeSign Identity Client`.

File: Casks/safesign-driver.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  homepage 'https://www.aeteurope.com/our-solutions/safesign-identity-client/'
+  license :gratis
+
+  pkg "SafeSignIC#{version.no_dots}-x86_64-MAC.pkg"
+
+  uninstall pkgutil: [
```
Comment (vitorgalvao) [inline Casks/safesign-driver.rb]: See https://github.com/caskroom/homebrew-cask/pull/25158/files. Did you miss `'SafeSign3.0'`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/safesign-driver.rb (hunk 1 above)
Comment (berniechiu) [inline Casks/safesign-driver.rb]: Sure~

File: Casks/safesign-driver.rb (hunk 2 above)
Comment (berniechiu) [inline Casks/safesign-driver.rb]: No problem

**Comment by rogeriopradoj** [comment]
@berniechiu @vitorgalvao 😃

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you. Merging.

**Pull request merged**